src/pages/getting-started/hello-world.mdx

```diff
@@ -270,7 +281,17 @@ async function lock(lovelace, { into, owner }): Promise<TxHash>{
 }
 ```
 
-Now is a good moment to pause and have a look at CardanoScan. Here's [an example of an _Hello World_ transaction](https://preview.cardanoscan.io/transaction/8559f57234407204d8e9a6bf57ef6943c65ec7119eb1c2ca6224f8bad8e71c1e?tab=utxo) that we generated using this tutorial.
+<Callout>
+  You can test the now updated code above live by submitting a new transaction to the blockchain, just run:
+
+  ```
+  deno run --allow-net --allow-read hello_world-lock.ts
+  ```
+
+  You should receive a response in a few seconds containing your `TxID` and `Datum`.
+</Callout>
```

Comment:
Hmm. This is a bit redundant with the callout few paragraphs up. But admittedly, it's the callout above that is misplaced and would be better down here here.  May you be kind enough and _move_ the Callout L212 here instead? Thanks.